File: java examples/Stateful Subsystem/java/org/team1085/guppy/commands/CloseGripper.java
package org.team1085.guppy.commands;

import edu.wpi.first.wpilibj.command.Command;
import org.team1085.bluegill.Robot;
import org.team1085.bluegill.Map;

public class CloseGripper extends Command {

  public ArcadeDrive () { requires(Robot.gripper); }
  protected void execute () { Robot.gripper.closeGripper(); }
  protected boolean isFinished () { return true; }
  
}

File: kotlin exampes/Stateful Subsystem/kotlin/org/team1085/guppy/commands/CloseGripper.kt
package org.team1085.guppy.commands

import edu.wpi.first.wpilibj.command.Command
import org.team1085.bluegill.Robot

class CloseGripper : Command () {

  init { requires(Robot.gripper) }

  override fun isFinished () = true
  override fun execute () = Robot.gripper.closeGripper()
  
}